api/src/main/resources/liquibase.xml

```diff
@@ -298,27 +298,33 @@
 	<changeSet id="ugandaemr-08022021-7" author="slubwama">
```

Comment:
Since the change set has some changes you need to change the ID of the changeset

Reply:
@Mwanje  also add a ticket to this PR.